src/quo/components/drawers/drawer_action/view.cljs

```diff
@@ -32,50 +34,59 @@
                                              :description?        (not-empty description)
                                              :blur?               blur?})
       :accessibility-label accessibility-label}
-     (when icon
-       [icon/icon icon
-        {:accessibility-label :left-icon
-         :container-style     (style/left-icon)
-         :color               (style/icon-color {:theme theme
-                                                 :type  action-type
-                                                 :blur? blur?})}])
-
      [rn/view
-      {:style (style/text-container)}
-      [text/text
-       (style/text {:theme theme
-                    :type  action-type
-                    :blur? blur?})
-       title]
+      {:style {:flex-direction :row
+               :align-items    :center}}
+      (when icon
+        [icon/icon icon
+         {:accessibility-label :left-icon
+          :container-style     (style/left-icon)
```

Comment:
Seems this was already like this, but I think  left-icon style should not be a function